src/diffpy/fourigui/__init__.py

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env python
 ##############################################################################
 #
-# diffpy.fourigui   by DANSE Diffraction group
-#                   Simon J. L. Billinge
-#                   (c) 2008 Trustees of the Columbia University
-#                   in the City of New York.  All rights reserved.
+# (c) 2024 The Trustees of Columbia University in the City of New York.
+# All rights reserved.
 #
-# File coded by:    Sani Harouna-Mayer
```

Comment:
We have to merge these headers more carefully. It looks as if Sani didn't do it correctly actually so there is some thinking to do here, but in general don't remove existing authors, but do add the "billinge group" catchall.  Then never remove previous copyright statements, but do add 2024 trustees of columbia....

Here I think the 2008 copyright statement is not right, so it would just be columbia copyright but with the right date range (please see when the the first commits were).

Reply:
another thing, let's do the headers and the functionality on different PRs.  Perhaps a global copy-replace to add 2024 columbia and group athors can work?